pyproject.toml

```diff
@@ -1,70 +1,102 @@
-[tool.poetry]
-name = "cluecoins"
-version = "0.0.0"
-description = "A CLI tool to manage the database of Bluecoins, an awesome budget planner for Android"
-license = "MIT"
-authors = [
-    "Lev Gorodetskiy <github@droserasprout.space>"
```

Comment:
😭

Reply:
Ahh, I got you back in the author's block